lightning/src/util/events.rs

```diff
@@ -152,6 +152,50 @@ impl_writeable_tlv_based_enum_upgradable!(ClosureReason,
 	(12, OutdatedChannelManager) => {},
 );
 
+/// Intended destination of a failed HTLC as indicated in [`Event::HTLCHandlingFailed`].
+#[derive(Clone, Debug, PartialEq)]
+pub enum HTLCDestination {
+	/// We tried forwarding to a channel but failed to do so. An example of such an instance is when
+	/// there is insufficient capacity in our outbound channel.
+	NextHopChannel {
+		/// The `node_id` of the next node. For backwards compatibility, this field is
+		/// marked as optional, since prior versions may not always be able to provide
+		/// counterparty node information.
```

Comment:
Should we say prior to which version?

Reply:
I updated the docs to put `0.0.110` in here. hope to land this PR for the upcoming release :)